src/types.rs

```diff
@@ -120,6 +120,7 @@ pub struct State {
     pub diagnosticsEnable: bool,
     pub diagnosticsList: DiagnosticsList,
     pub diagnosticsDisplay: HashMap<u64, DiagnosticsDisplay>,
+    pub diagnosticsMaxSigns: i64,
```

Comment:
Please use Option for this setting, i.e., `Option<u64>`. When the option is not specified, it should be none.

Reply:
I updated the type to `Option<u64>`.